crates/ironrdp-pdu/src/rdp/vc/dvc.rs

```diff
@@ -11,6 +11,7 @@ use crate::PduParsing;
 #[cfg(test)]
 mod tests;
 
+pub mod ainput;
```

Comment:
thought: Ideally, I would like to move code for specific channels in their own crates. For instance, we do this for the RDPDR static channel, for which the code is found in the `ironrdp-rdpdr` crate. In this case, a `ironrdp-ainput` crate could be created. It’s fine to do a follow-up PR for that though.

Reply:
done